add mt84 keyboard (#10857)
qmk/qmk_firmware: keyboards/mt84/readme.md

# Mt84rgb 

![Mt84rgb](https://i.imgur.com/2fJg3DD.png)

A 84% keyboard PCB made by Mt.
It supports standard 84 keyboard，with hotswap and rgbbacklight

* Keyboard Maintainer: [MT](https://github.com/704340378)
* Hardware Supported: mt84rgb (atmega32u4)
* Hardware Availability: [麦田外设](https://shop110310565.taobao.com)

Make example for this keyboard (after setting up your build environment):

>     make mt84:default

Reviewer: ```suggestion
    make mt84:default

Flashing example for this keyboard:

    make mt84:default:flash

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

# Mt84rgb 

> ![Mt84rgb](https://i.imgur.com/2fJg3DD.png)

Reviewer 2: Swaps this image for a smaller version.

```suggestion
![Mt84rgb](https://i.imgur.com/2fJg3DDl.png)
```

> A 84% keyboard PCB made by Mt.
> It supports standard 84 keyboard，with hotswap and rgbbacklight

Reviewer 2: Just some grammatical cleanup.

```suggestion
A 84% keyboard PCB made by MT.
It supports standard 84-key layout, with hotswap sockets and per-key RGB backlight.
```

* Keyboard Maintainer: [MT](https://github.com/704340378)
* Hardware Supported: mt84rgb (atmega32u4)
* Hardware Availability: [麦田外设](https://shop110310565.taobao.com)

Make example for this keyboard (after setting up your build environment):

    make mt84:default
Flashing example for this keyboard:
    make mt84:default:flash

> See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

Reviewer: ```suggestion

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

```

# Mt84rgb 

![Mt84rgb](https://i.imgur.com/2fJg3DDl.png)

A 84% keyboard PCB made by MT.
It supports standard 84-key layout, with hotswap sockets and per-key RGB backlight.

* Keyboard Maintainer: [MT](https://github.com/704340378)
* Hardware Supported: mt84rgb (atmega32u4)
* Hardware Availability: [麦田外设](https://shop110310565.taobao.com)

> Make example for this keyboard (after setting up your build environment):
> 
>     make mt84:default
> Flashing example for this keyboard:
>     make mt84:default:flash

Reviewer: ```suggestion
Make example for this keyboard (after setting up your build environment):

    make mt84:default

Flashing example for this keyboard:

    make mt84:default:flash
```

# Mt84rgb 

![Mt84rgb](https://i.imgur.com/2fJg3DDl.png)

A 84% keyboard PCB made by MT.
It supports standard 84-key layout, with hotswap sockets and per-key RGB backlight.

* Keyboard Maintainer: [MT](https://github.com/704340378)
* Hardware Supported: mt84rgb (atmega32u4)
* Hardware Availability: [麦田外设](https://shop110310565.taobao.com)

Make example for this keyboard (after setting up your build environment):

    make mt84:default
    
Flashing example for this keyboard:

    make mt84:default:flash

> 

Reviewer 3: Can you add "clear instructions on how to reset the board into bootloader mode" as per the [PR checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?

Author: press the reset key will let it into bootloader.
sorry i dont know what line should i add in the readme file ,i had check other keyboads and no find this instructions in there readme.md
can you give the line i will add it in ,thank you very much

Author: thank you very much

> 

Author: thanks
